gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreWrapper.java

```diff
@@ -189,7 +189,7 @@ public Entity read(Key key) {
      * @return results fo the query packed in a {@link List}
      * @see DatastoreReader#run(Query)
      */
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings("unchecked") // Untyped query which always works with datastore {@code Entity}.
```

Comment:
Please rephrase to make it more user-friendly. Remember, that's an explanation, not a comment to the line where you have that `query` declared.
Also, `datastore` should be a product name.